core/commands/storage/challenge/challenge.go

```diff
@@ -138,11 +150,17 @@ the challenge request back to the caller.`,
 		if err != nil {
 			return err
 		}
+		res.RecordEvent("HNewResponse")
+
 		err = sc.SolveChallenge(chunkIndex, nonce)
 		if err != nil {
 			return err
 		}
-		return cmds.EmitOnce(res, &StorageChallengeRes{Answer: sc.Hash})
+		res.RecordEvent("HSolveChallenge")
+		return cmds.EmitOnce(res, &StorageChallengeRes{
+			Answer:       sc.Hash,
+			TimeEvaluate: []string{res.ShowEventReport()},
```

Comment:
change it to string? since report is now always string? unless you want to make another []string version

Reply:
actually in the storageChallengeRequestCmd need use the TimeEvaluate as []string to record both the request time and the response time.

Reply:
got it